magda-web-client/src/Components/Header/Header.js

```diff
@@ -17,13 +17,18 @@ class Header extends Component {
     }
 
     componentDidMount() {
+        this._isMounted = true;
```

Comment:
I don't think you interact with this property later?
Sometimes, people define a local `_isMounted ` state to maintain the mount/unmount state.
But  you need to check this property before the problematic `setState` somewhere?

Reply:
Good catch! Added the check